app/controllers/idv/in_person/verify_info_controller.rb

```diff
@@ -95,6 +96,12 @@ def confirm_ssn_step_complete
         return if pii.present? && idv_session.ssn.present?
         redirect_to prev_url
       end
+
+      def confirm_pii_data_present
+        unless user_session.dig('idv/in_person').present?
+          redirect_to idv_path
+        end
```

Comment:
oh I missed this in the original PR but I think it would better with a positive check, and maybe even fit on one line if we want
```suggestion
        redirect_to idv_path if user_session.dig('idv/in_person').blank?
```